hardware/raspberrypi/bootmodes/net_tutorial.md

```diff
@@ -35,6 +35,18 @@ Ensure the output `0x3020000a` is correct.
 
 The client configuration is almost done. The final thing to do is to remove the `program_usb_boot_mode` line from `config.txt` (make sure there is no blank line at the end). You can do this with `sudo nano /boot/config.txt`, for example. Finally, shut the client Raspberry Pi down with `sudo poweroff`.
 
+## Client configuration (Raspberry Pi 3B+ / 4)
+
+These instructions only apply to the Raspberry Pi 3B+ and 4.
+
+Follow the instructions on the [Pi 4 Bootloader Configuration](../bcm2711_bootloader_config.md) page to alter the boot order to include network booting.  In short:
+
+```bash
+sudo -E rpi-eeprom-config --edit
+```
+
+Set `BOOT_ORDER` to `0xf21` or similar, save, and reboot.  The updated EEPROM settings will be saved onto the SD card and programmed to the EEPROM during the next boot.  After this reboot the settings will become permanent and the SD card is no longer required.
```

Comment:
I think there might be a `raspi-config` option to do this now, which is easier than using `rpi-eeprom-config` directly?

Ah yes: https://www.raspberrypi.org/documentation/configuration/raspi-config.md